main.go

```diff
@@ -72,6 +73,19 @@ func main() {
 	mongoDBAuth := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_AUTH", true, true)
 	mongoDBName := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_DATABASE", true, false)
 	mongoTimeout := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_TIMEOUT", false, false)
+	// webhook configs
+	githubWebhookRequestToken := envLoader.GetAndLogEnvVar("GITHUB_WEBHOOK_REQUEST_TOKEN", false, false)
+	githubToken := envLoader.GetAndLogEnvVar("GITHUB_TOKEN", false, false)
+	githubOrgnizationName := envLoader.GetAndLogEnvVar("GITHUB_ORG_NAME", false, false)
```

Comment:
We should rename all of these environment variables to have the `GITHUB_APP_CONFIG_` prefix instead of just `GITHUB_`. We can keep `GITHUB_TOKEN` as is since we should be able to use the same token for any future GitHub integrations as well.